config/objects/moddables.json

```diff
@@ -115,9 +115,38 @@
 			}
 		},
 		"types" : {
-			"evil" : { "index" : 0,	"rmg" : { "mapLimit" : 64 } },
-			"good" : { "index" : 1, "rmg" : { "mapLimit" : 64 } },
-			"neutral" : { "index" : 2, "rmg" : { "mapLimit" : 64 } },
+			"evil" : 
+			{ 
+				"index" : 0,	
+				"rmg" : { "mapLimit" : 64 },
```

Comment:
RMG threats Boat as a special object and doesn't really check this limit. Engine doesn't have such limitations either, afaik.